File: An 3/Sem 2/MPP/Java-CSharp/Server/src/main/java/com/mcostea/SalesAgency/model/Product.java
package com.mcostea.SalesAgency.model;

import java.io.Serializable;

/**
 * Created by mihaicostea on 16/04/15.
 */
public class Product implements Serializable {
    private int productId;
    private String name;
    private int price;
    private int quantity;

    public int getProductId() {
        return productId;
    }

    public void setProductId(int productId) {
        this.productId = productId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: An 3/Sem 2/MPP/Java-CSharp/Client/Model/Product.cs
using System;

namespace Model
{
	[Serializable]
	public class Product
	{

		public int ProductId { get; set; }
		public String Name { get; set; }
		public int Price { get; set; }
		public int Quantity { get; set; }

		public Product ()
		{
		}
	}
}

